src/ifcblenderexport/blenderbim/bim/module/bimtester/ui.py

```diff
@@ -0,0 +1,61 @@
+import bpy
+from bpy.types import Panel
+
+
+class BIM_PT_qa(Panel):
+    bl_label = "BIMTester Quality Auditing"
+    bl_idname = "BIM_PT_qa"
+    bl_options = {"DEFAULT_CLOSED"}
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "scene"
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+
+        scene = context.scene
+        bimtester_properties = bpy.context.scene.BimTesterProperties
+
+        layout.label(text="Gherkin Setup:")
+
+        row = layout.row(align=True)
+        row.prop(bimtester_properties, "features_dir")
+        row.operator("bim.select_features_dir", icon="FILE_FOLDER", text="")
+
+        if bimtester_properties.features_dir:
+            row = layout.row(align=True)
+            row.prop(bimtester_properties, "features_file")
+
+            row = layout.row(align=True)
+            row.prop(bimtester_properties, "scenario")
+        else:
+            return
+
+        if context.scene.BimTesterProperties.features_file != '': # To handle the error when no .feature file exists in the folder
```

Comment:
Here, I need to access the STRING value of the EnumProperty, knowing that if nothing is selected it might return a 0 ...
What should I do in this case as I do not know all the EnumProperty Methods

Reply:
Code looks good. There actually are no enumproperty methods. You only get access to the value of the currently selected enums.